### DISCUSSION TITLE
Calibrate CSS: links and use cases

### DISCUSSION THREAD

**Discussion by graphicore** [graphicore/varla-varfo#9] [Show and tell]
Hi,

I'm looking for [Links of Implementations](https://graphicore.github.io/varla-varfo/explorations/calibrate/#links) and [Use Cases](https://graphicore.github.io/varla-varfo/explorations/calibrate/#use-cases) for the [Calibrate CSS article](https://graphicore.github.io/varla-varfo/explorations/calibrate/).

**Comment by davelab6** [comment]
What is the status of this?

**Comment by graphicore** [comment]
Here's a use case:

Suppose I want to practice Darts at home. I don't want to hang a physical dart board nor throw pointy darts around (maybe I have kids or pets). I have some kind of special dart:
* With a rubber point that will be registered by my touchscreen, but not break it.
* With acceleration sensors and realistic size/weight, it will be used to analyze and simulate my throw in software. This devise is attached to my wrist or the ceiling with a leash and it will not hit the screen/wall/ground.

The latter dart, given appropriate software and good accuracy of the sensor could be used for high level training of sophisticated athletes. There's also the opportunity for online/virtual training/competition platforms, like Zwift for running and cycling.

All I need else is a real world sized dartboard on my screen (which is mounted at the right height or I use a projector) and enough space in front of the screen ( 2.37 m plus me).
